docs/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -57,19 +55,102 @@ This section considers all public endpoints existing in the component REST APIs
 - Other attributes of entities themselves have no prefix to avoid clutter, for example version in the process definition entity. In other resources, they have to be referenced with a prefix, like `processDefinitionVersion` in the process instance entity.
-- The `bpmnProcessId` is now called `processDefinitionId` to be easily relatable to the entity (process definition) and the accompanying `processDefinitionKey`.
-- The `decisionKey` and `dmnDecisionKey` are now aligned to `decisionDefinitionKey`, the `decisionId` and `dmnDecisionId` to `decisionDefinitionId`. Similar to the `processDefinitionId`, those attributes are now related to the entity `decisionDefinition`.
+- The `bpmnProcessId` and `processName` are now called `processDefinitionId` to be easily relatable to the process definition entity, like the `processDefinitionKey`.
+- The `decisionKey` and `dmnDecisionKey` are now aligned to `decisionDefinitionKey`, the `decisionId` and `dmnDecisionId` to `decisionDefinitionId`. Similar to the `processDefinitionId` being related to the process definition, those attributes are now easily relatable to the decision definition entity.
 
 <!--- Insert Operate section with V1 endpoint and V2 endpoint to use with input/output adjustments --->
 
-### Tasklist
+## Tasklist API
+
+### Form
+
+#### Get a form
+
+- **V1 endpoint**: `GET /v1/forms/{formId}`
+- **V2 endpoints**:
+  - `GET /v2/user-tasks/{userTaskKey}/form`
+  - `GET /v2/process-definitions/{processDefinitionKey}/form`
+
+<Tabs groupId="get-form" defaultValue="input-adjustments" queryString values={
+[
+{label: 'Input adjustments', value: 'input-adjustments'},
+{label: 'Output adjustments', value: 'output-adjustments'},
+]
+}>
+
+<TabItem value='input-adjustments'>
+
+- You cannot fetch forms directly anymore. Instead, fetch them by user task or process definition to get the respective form data.
+- The respective endpoint only takes the key of the resource the form is related to as input parameter.
+
+</TabItem>
+
+<TabItem value='output-adjustments'>
+
+- Embedded forms are no longer returned as Camunda user tasks don't support them.
+- Renamed atttibutes
+  - `id` - Use `formKey` as this refers to the unique system identifier of the form.
+  - `title` - Use `formId` as this aligns better with the attribute defined in the form schema.
+- Removed attributes
+  - `isDeleted` - The endpoint does not serve this information anymore.
+  - `processDefinitionKey` - You can identify the related entity from the endpoint resource and the provided key parameter, the form response does not contain it additionally anymore.
+
+</TabItem>
+</Tabs>
+
+### Task
+
+#### Save task draft variables
+
+- **V1 endpoint**: `POST /v1/tasks/{taskId}/variables`
+- **V2 endpoint**: This feature is not supported in V2 anymore. Use [setting variables][] as `local` to the user task's `elementInstanceKey` as a replacement.
```

Comment:
I think we have an accidental missing link here, or is this documentation not available?